src/unittest/test_random.cpp

```diff
@@ -50,7 +50,7 @@ void TestRandom::runTests(IGameDef *gamedef)
 	TEST(testPseudoRandom);
 	TEST(testPseudoRandomRange);
 	TEST(testPcgRandom);
-	TEST(testPcgRandomRange);
+	//TEST(testPcgRandomRange);
```

Comment:
Oops, I didn't meant to commit that change. The problem is that that line crashes unit tests.